packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -1270,7 +1274,10 @@ export class ApolloServerBase {
    * `{req: express.Request, res: express.Response }` object) and to keep it
    * updated as you upgrade Apollo Server.
    */
-  public async executeOperation(request: GraphQLRequest, integrationContextArgument?: Record<string, any>) {
+  public async executeOperation(
```

Comment:
I've been keeping this particular file prettier-happy but failed to do so on a PR I merged today, sigh.

Reply:
eh I'll roll this part of the diff back. since I do want to cherry-pick this on top of the last release.